less/kikar.less

```diff
@@ -22,75 +22,73 @@
   color: #464545;
 }
 
-#loading-statuses-symbol{
+#loading-statuses-symbol {
   text-align: center;
   display: none;
 }
 
-
 /*  Styling for responsive embedding by http://embedresponsively.com/*/
 
 .embed-container {
-    position: relative;
-    padding-bottom: 56.25%;
-    height: 0;
-    overflow: hidden;
-    max-width: 100%;
-}
-
-.embed-container iframe, .embed-container object, .embed-container embed {
+  position: relative;
+  padding-bottom: 56.25%;
+  height: 0;
+  overflow: hidden;
+  max-width: 100%;
+  iframe object embed {
     position: absolute;
     top: 0;
     left: 0;
     width: 100%;
     height: 100%;
+  }
 }
- /*  Styles for _facebook_status_section_update_elem.html */
+
+/*  Styles for _facebook_status_section_update_elem.html */
 .fb-attachment-container {
-    text-align: center;
-    border-style: solid none none none;
-    border-top-color: #dddddd;
-    border-top-width: thin;
+  text-align: center;
+  border-style: solid none none none;
+  border-top-color: #dddddd;
+  border-top-width: thin;
 }
 
 .img-fb-status-attachment {
-    height: auto;
-    margin: auto;
-    border: solid;
-    border-width: thin;
-    border-color: #dddddd;
-}
+  height: auto;
+  margin: auto;
+  border: solid;
+  border-width: thin;
+  border-color: #dddddd;
 
-@media (min-width: 768px) {
+  @media (min-width: 768px) {
     .img-fb-status-attachment {
```

Comment:
You don't need the double class inside, you already get it from the outside.. maybe that's why the classes are doubled?